Classes/Command/CreateTestDataCommand.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Tea\Command;
+
+/*
+ * Command to create test data for the tea extension.
+ */
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use TYPO3\CMS\Core\Database\Connection;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+final class CreateTestDataCommand extends Command
+{
+    /** @phpstan-ignore-next-line */
+    protected array $teaData = [
+        [
+            'title' => 'Darjeeling',
+            'description' => 'I love that tea!',
+            'sys_language_uid' => 0,
+        ],
+         [
+            'title' => 'Earl Grey',
+            'description' => 'A nice tea!',
+            'sys_language_uid' => 0,
+        ],
+    ];
+    protected function configure(): void
+    {
+        $this
+            ->setHelp('Create test data for the tea extension in an already existing page (sysfolder).')
+            ->addArgument(
+                'pageId',
+                InputArgument::REQUIRED,
+                'Existing sysfolder page id.'
+            )
+            ->addOption(
+                'delete-data-before',
+                'd',
+                InputOption::VALUE_NONE,
+                'Delete all tea data in the defined pid before creating new data.'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        /** @var int $pageId */
```

Comment:
In production code, please let's use `assert` to state data restrictions that we (as human) know to be true.